src/suites/cts/format_info.ts

```diff
@@ -62,4 +55,22 @@ export const textureFormatInfo: TextureFormats = {
   'depth24plus-stencil8':   { renderable:  true, color: false },
 };
 export const textureFormats = Object.keys(textureFormatInfo) as GPUTextureFormat[];
-export const textureFormatParams = Array.from(poptions('format', textureFormats));
+
+export const bindingTypeInfo: {
+  [k in GPUBindingType]: {
+    type: 'buffer' | 'texture' | 'sampler';
+    maxDynamicCount: number;
+    // Add fields as needed
+  };
+} = /* prettier-ignore */ {
+  'uniform-buffer':          { type: 'buffer',  maxDynamicCount: 8 },
```

Comment:
Can this be like `[C.BindingType.UniformBuffer]: { ... }` ?

Reply:
Could, but I don't think it's helpful. These keys are already tightly typed by `[k in GPUBindingType]` so typos aren't possible. In fact, that forces us to be complete too (no missing variants). Also the original string values are shorter and I think easier to read.